Bandwidth.Iris/Client.cs

```diff
@@ -396,51 +397,50 @@ private static string FixPath(string path)
 
         private async Task CheckResponse(HttpResponseMessage response)
         {
-            try
+            if (!response.IsSuccessStatusCode)
```

Comment:
You can avoid the additional nesting with a return at the top.

`if (response.IsSuccessStatusCode)
 return;`